inst/unitTests/cpp/misc.cpp

```diff
@@ -179,22 +179,22 @@ void test_stop_variadic() {
 }
 
 // [[Rcpp::export]]
-bool testNullableForNull(Nullable<NumericMatrix> M) {
+bool testNullableForNull(const Nullable<NumericMatrix>& M) {
```

Comment:
It's no big deal to re-add the pre-PR versions, but I wonder under what circumstances code can succeed for `const&` but fail for a non-const object?